VRCModLoader.csproj

```diff
@@ -45,101 +49,101 @@
     </Reference>
     <Reference Include="Assembly-CSharp">
       <HintPath>..\..\..\..\jeux\steam\SteamApps\common\VRChat\VRChat_Data\Managed\Assembly-CSharp.dll</HintPath>
-      <HintPath>lib\Assembly-CSharp.dll</HintPath>
+      <HintPath>D:\Steam Library\steamapps\common\VRChat\VRChat_Data\Managed\Assembly-CSharp.dll</HintPath>
```

Comment:
Just NO

Reply:
References weren't intended to be included anyway, my bad